core/domain/user_jobs_one_off.py

```diff
@@ -87,6 +87,29 @@ def reduce(key, stringified_username_counter):
         yield (key, len(username_counter))
 
 
+class UserBioLengthOneOffJob(jobs.BaseMapReduceJobManager):
+    """One-off job for calculating the length of user_bio.
+
+    Returns:
+        None: If all the user_bios' length is None or less than 500 chars.
+        Lists: List of length of user_bio, user_name  for char
+            greater than 500.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [user_models.UserSettingsModel]
+
+    @staticmethod
+    def map(item):
+        yield (len(item.user_bio), item.username)
+
+    @staticmethod
+    def reduce(key, username):
```

Comment:
I think the second arg would be a stringified list of usernames. Can you check (and compare with other jobs)?